azure-kusto-data/src/operations/mgmt.rs

```diff
@@ -0,0 +1,155 @@
+use crate::client::KustoClient;
+use crate::models::{QueryBody, TableV1};
+use async_convert::TryFrom;
+use azure_core::prelude::*;
+use azure_core::setters;
+use azure_core::{collect_pinned_stream, Response as HttpResponse};
+use futures::future::BoxFuture;
+use serde::{Deserialize, Serialize};
+
+type ManagementQuery = BoxFuture<'static, crate::error::Result<KustoResponseDataSetV1>>;
+
+#[derive(Debug, Clone)]
+pub struct ManagementQueryBuilder {
```

Comment:
Most of the code here is copy-pasted from query.rs.
I'm pretty sure we can do it in one generic class (I.E type ManagementQueryBuilder = QueryBuilder<ManagementQuery>)
Or at worst, having most of the functionality in one type, and have the QueryBuilders be thin wrappers around it

Reply:
Agreed, I tried doing it with asyc-convert, but there were some issues https://github.com/Azure/azure-sdk-for-rust/pull/685#discussion_r835772356 but there certainly is a way... wanted to leave this as a future optimization, but can try to tackle it now as well.